pyaedt/edb_core/layout.py

```diff
@@ -928,6 +938,20 @@ def _validatePoint(self, point, allowArcs=True):
                 self._logger.error("Point Y value must be a number.")
                 return False
             return True
+        elif len(point) == 3:
+            if not allowArcs:
+                self._logger.error("Arc found but arcs are not allowed in _validatePoint.")
+                return False
+            if not isinstance(point[0], (int, float, str)):
+                self._logger.error("Point X value must be a number.")
+                return False
+            if not isinstance(point[1], (int, float, str)):
+                self._logger.error("Point Y value must be a number.")
+                return False
+            if not isinstance(point[1], (int, float)):
```

Comment:
Why `z` cannot be a `str` like `x` and `y`?
```suggestion
            if not isinstance(point[1], (int, float,str)):
```